README.md

```diff
@@ -1,2 +1,68 @@
-# module1-capstone
- HTML + CSS + JS project to implement concert or conference web site.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Module 1 - Capstone Project
+
+HTML + CSS + JS project to implement International Conference on Open-Water Swimming conference web site.
+
+The website it's Mobile-first designed and have breakpoints for 768px and less, between 768px and 1024px, and 1024px and more screen size devices.
+
+The functionality available is the Landing Page and the About Page.
+
+For the landing page, the sections to display the program topics and the relevant speakers are built dynamically by reading the info from data files in JavaScript.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- Bootstrap CSS Framework for styling.
+
+## Live Demo
+
+https://lmunozm1702.github.io/module1-capstone/
+
+## Getting Started
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+Once on the landing page, you can scroll down to see the general info and click the about link on the menu (select the hamburger menu first if you are connected from a mobile device), to see the organization info.
+
+### Run tests
+
+### Deployment
+
+The project is deployed into Github Pages service.
+
+## Authors
+
+### 👤 Luis Muñoz
+
+- GitHub: [@lmunozm1702](https://github.com/lmunozm1702)
+- Twitter: [@lmunozm](https://twitter.com/lmunozm)
+- LinkedIn: [luis-munoz-manriquez](https://www.linkedin.com/in/luis-munoz-manriquez)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+The design of the website is inspired by the idea of [Cindy Shin](https://www.behance.net/adagio07), and publicated under license of CreativeCommons.
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] I think it would be better to remove this `_NOTE` from your README file 💯

> We should always be sure our repositories are documented professionally, please fix broken links on the readme file.